tensorflow/core/util/gpu_kernel_helper_test.cu.cc

```diff
@@ -318,6 +332,9 @@ TEST(CudaDeviceFunctionsTest, ShuffleGetSrcLane) {
   ASSERT_EQ(hipHostMalloc(&failure_count, sizeof(unsigned), 0), cudaSuccess);
 #endif
   *failure_count = 0;
+#if TENSORFLOW_USE_ROCM
+  const int TF_RED_WARPSIZE=isGfx10() ? 32 : 64;
```

Comment:
It's a bit ugly how it's either macro or a variable. Can we just make it a variable in both cases, and change the naming convention?